robert_gazebo/models/fire_rescue/model.sdf

```diff
@@ -0,0 +1,38 @@
+<?xml version='1.0'?>
+<sdf version='1.5'>
+    <model name="fire_rescue_box">
+        <pose>0 0 0 0 0 0</pose>
+        <static>true</static>
```

Comment:
in order to have interaction with the object in simulation please set this property to false